mmv1/third_party/terraform/services/compute/resource_compute_instance_from_machine_image.go.tmpl

```diff
@@ -64,6 +64,39 @@ func computeInstanceFromMachineImageSchema() map[string]*schema.Schema {
 		ForceNew:    true,
 		Description: `Name or self link of a machine image to create the instance from on.`,
 	}
+	s["source_machine_image_encryption_key"] = &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Description: `Optional encryption key fields for the source machine image." + 
+			"Supported fields: raw_key, rsa_encrypted_key, kms_key_name, sha256, kms_key_service_account."`,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"raw_key": {
+					Type:      schema.TypeString,
+					Optional:  true,
+					Sensitive: true,
+				},
+				"rsa_encrypted_key": {
+					Type:      schema.TypeString,
+					Optional:  true,
+					Sensitive: true,
+				},
+				"kms_key_name": {
+					Type:     schema.TypeString,
+					Optional: true,
+				},
+				"sha256": {
```

Comment:
Looks like this shouldn't be settable by the user. It should be marked Computed: true